src/ontology/mappings/mondo_closematch_meddra.sssom.tsv

```diff
@@ -7,11 +7,9 @@
 #   semapv: https://w3id.org/semapv/vocab/
 #   skos: http://www.w3.org/2004/02/skos/core#
 #   sssom: https://w3id.org/sssom/
-# license: https://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
Why is this using the wrong license now?

Reply:
Fixed